### DISCUSSION TITLE
How to get midi in channel as number?

### DISCUSSION THREAD

**Discussion by huykaamaa** [tttapa/Control-Surface#939] [Q&A] [answered]
Hello, i want to get midi channel from DAW as number 1-16, but i don’t know how to :(
I used input exam but it show “CHANNEL_1” or “Channel 1”

**Comment by tttapa** [comment, marked as answer]
https://tttapa.github.io/Control-Surface-doc/Doxygen/d2/d48/classChannel.html#a8d3578ee58408eb7f1e264cd9a6a7423

> I used input exam but it show “CHANNEL_1” or “Channel 1”

What do you mean?

**Reply by huykaamaa** [reply]
Thank you, sir
getOneBased() work well
